tests/test_schema_validation.py

```diff
@@ -601,6 +601,15 @@ def test_invalid_repeating_section_list_name():
     check_validation_errors(filename, expected_error_messages)
 
 
+def test_invalid_repeating_section_title_placeholders():
+    filename = 'schemas/invalid/test_invalid_repeating_section_title_placeholders.json'
+    expected_error_messages = [
+        "Schema Integrity Error. Placeholders in 'text' doesn't match 'placeholders' definition for block id 'None'",
```

Comment:
Why is this None?

Reply:
I've changed this to include the placeholder text and the missing placeholder definition